src/index.ts

```diff
@@ -0,0 +1,41 @@
+import express, {Application} from "express";
+import cors from "cors";
+import { sortListByBlocks } from "./blocks";
+import {handler} from "./handler";
+
+/*
+Proxy/Load balancer
+*/
+
+const SORT_DELAY_IN_MINUTES: number = 1;
+
+// App port
+const port = process.env.PORT || 8080 
+
+const app: Application = express()
+
+// Remove the X-Powered-By headers.
+app.disable('x-powered-by');  
+
+app.use(cors());
```

Comment:
We should not use cors protection as we will be accessing this resource from multiple domains